publication/tests/test_publication.py

```diff
@@ -135,24 +135,24 @@ def test_distribution_list(self):
         assigned_count = \
             distribution_model.get_product_contract_assigned_count(
                 product_newsletter.id, partner_jan.id)
-        self.assertEqual(assigned_count, 0)
+        self.assertEqual(assigned_count, 1)
         subscription01 = distribution_model.create({
             'product_id': product_newsletter.id,
             'contract_partner_id': partner_jan.id,
             'partner_id': partner_joris.id,
             'copies': 10})
         self.assertEqual(subscription01.contract_count, 40)
-        self.assertEqual(subscription01.assigned_count, 10)
-        self.assertEqual(subscription01.available_count, 30)
+        self.assertEqual(subscription01.assigned_count, 11)
+        self.assertEqual(subscription01.available_count, 29)
         subscription02 = distribution_model.create({
             'product_id': product_newsletter.id,
             'contract_partner_id': partner_jan.id,
             'partner_id': partner_corneel.id,
             'copies': 10})
         self.assertEqual(subscription02.contract_count, 40)
         # Old entry should also be changed
-        self.assertEqual(subscription01.assigned_count, 20)
-        self.assertEqual(subscription01.available_count, 20)
+        self.assertEqual(subscription01.assigned_count, 21)
```

Comment:
Same here for assigned and available. These values lose some of their meaning now that we always send all copies that are not send to other partners to the contract partner. Only if a manual update is made, when a contract partner does not want to receive all copies he paid for can this still be different.